### PR TITLE
Add Running tests section on CONTRIBUTING

### PR DESCRIPTION
**Pull request by maurogeorge**

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,3 +29,23 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
 7. Create a new Pull Request
 
 Remember to squash your commits and rebase off `master`.
+
+### Running tests
```
Comment (bf4) [inline CONTRIBUTING.md]: This should go, I think, in the README.. @rails-api/ams thoughts?

Comment (bf4) [inline CONTRIBUTING.md]: btw, :+1: thanks for this

**Comment by beauby** [comment]
:+1:

**Comment by maurogeorge** [comment]
Thanks for the review.

This is good to merge or I need to move this to README? I think the CONTRIBUTING is a better place, I think most people that download and run the tests are people the are contributing or trying contribute. I think the usual use case is add to gemfile and bundle it.

File: CONTRIBUTING.md (hunk 1 above)
Comment (beauby) [inline CONTRIBUTING.md]: @bf4 Most users probably do not really care about actually running the tests I believe. What's your rationale behind adding it to README?

Comment (joaomdmoura) [inline CONTRIBUTING.md]: agree with @beauby ppl on readme are probably not looking into this :grin:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,3 +29,23 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
...
+
+#### Running with Rake
+
+The easiest way to run the unit tests is through Rake. The default task runs
+the entire test suite for all classes. For more information, checkout the
+full array of rake tasks with "rake -T"
+
+Rake can be found at http://docs.seattlerb.org/rake/.
+
+#### Running by hand
```
Comment (joaomdmoura) [inline CONTRIBUTING.md]: I would remove this subtitle

**Comment by joaomdmoura** [comment]
Nice work here @maurogeorge I'm sure this is help ppk trying jump in the project!

File: CONTRIBUTING.md (hunk 2 above)
Comment (beauby) [inline CONTRIBUTING.md]: Agreed. Let's remove this subtitle.

**Comment by maurogeorge** [comment]
Thanks for the review guys.

I updated the code :smile:

**Comment by joaomdmoura** [comment]
LGTM, merging it.

**Comment by maurogeorge** [comment]
Thanks :heart:

**Pull request merged**